CHANGELOG.md

```diff
@@ -1,3 +1,1038 @@
+
+- **fix(jqLite): align jqLite camelCasing logic with JQuery ([73050c](https://github.com/angular/angular.js/commit/73050cdda04675bfa6705dc841ddbbb6919eb048))**:
+
+Before, when Angular was used without jQuery, the key passed
+to the css method was more heavily camelCased; now only a single (!) hyphen
+followed by a lowercase letter is getting transformed. This also affects APIs
+that rely on the css method, like ngStyle.
+
+If you use Angular with jQuery, it already behaved in this way so no changes
+are needed on your part.
+
+To migrate the code follow the example below:
+
+Before:
+
+HTML:
+
+```
+// All five versions used to be equivalent.
+<div ng-style={background_color: 'blue'}></div>
+<div ng-style={'background:color': 'blue'}></div>
+<div ng-style={'background-color': 'blue'}></div>
+<div ng-style={'background--color': 'blue'}></div>
+<div ng-style={backgroundColor: 'blue'}></div>
+```
+
+JS:
+
+```
+// All five versions used to be equivalent.
+elem.css('background_color', 'blue');
+elem.css('background:color', 'blue');
+elem.css('background-color', 'blue');
+elem.css('background--color', 'blue');
+elem.css('backgroundColor', 'blue');
+
+// All five versions used to be equivalent.
+var bgColor = elem.css('background_color');
+var bgColor = elem.css('background:color');
+var bgColor = elem.css('background-color');
+var bgColor = elem.css('background--color');
+var bgColor = elem.css('backgroundColor');
+```
+
+After:
+
+HTML:
+
+```
+// Previous five versions are no longer equivalent but these two still are.
+<div ng-style={'background-color': 'blue'}></div>
+<div ng-style={backgroundColor: 'blue'}></div>
+```
+
+JS:
+
+```
+// Previous five versions are no longer equivalent but these two still are.
+elem.css('background-color', 'blue');
+elem.css('backgroundColor', 'blue');
+
+// Previous five versions are no longer equivalent but these two still are.
+var bgColor = elem.css('background-color');
+var bgColor = elem.css('backgroundColor');
+```
+
+- **feat(jqLite): don't remove a boolean attribute for `.attr(attrName, '')` ([3faf45](https://github.com/angular/angular.js/commit/3faf4505732758165083c9d21de71fa9b6983f4a))**:
+
+Before, using the `attr` method with an empty string as a value
+would remove the boolean attribute. Now it sets it to its lowercase name as
+was happening for every non-empty string so far. The only two values that remove
+the boolean attribute are now null &amp; false, just like in jQuery.
+
+To migrate the code follow the example below:
+
+Before:
+
+```
+elem.attr(booleanAttrName, '');
+```
+
+After:
+
+```
+elem.attr(booleanAttrName, false);
+```
+
+or:
+
+```
+elem.attr(booleanAttrName, null);
+```
+
+- **feat(jqLite): remove the attribute for `.attr(attribute, null)` ([4e3624](https://github.com/angular/angular.js/commit/4e3624552284d0e725bf6262b2e468cd2c7682fa))**:
+
+Invoking `elem.attr(attributeName, null)` would set the
+`attributeName` atribute value to a string `&quot;null&quot;`, now it removes the
```

Comment:
Right! I had a problem with nunjucks escaping stuff. The tool is fixed but I manually updated this file and didn't catch them all.